root/static/scripts/release-editor/edits.js

```diff
@@ -422,15 +421,19 @@ releaseEditor.getEditPreviews = function () {
         releaseEditor.editPreviews(_.compact(_.map(edits, getPreview)));
     }
 
-    function getPreview(edit) { return previews[edit.hash] }
+    function getPreview(edit) {
+        return previews[edit.hash];
+    }
```

Comment:
The previous preview is separated with a newline, so let's try and be consistent here too :) Same below (we could also remove all newlines, but I at least feel newlines make it more clear)